src/utils/disks/helpers.ts

```diff
@@ -47,10 +48,10 @@ export function getPDiskId({
     return undefined;
 }
 
-export function getVDiskSlotBasedId(
-    nodeId: string | number,
-    pDiskId: string | number,
-    vDiskSlotId: string | number,
-) {
-    return [nodeId, pDiskId, vDiskSlotId].join('-');
+export function getVDiskSlotBasedId(params: VDiskBlobIndexStatParams) {
```

Comment:
I'd rename it to something else. Maybe just `getVDiskId`, vdisk id and slot id are different:

Id: "GroupID - GroupGeneration - Ring - Domain - VDisk"
Slot id: "NodeId - PDiskId - VDiskSlotId"